recipes-extended/spi-minnowmax-board/files/modprobe.d/spi-minnow-at86rf230.conf.sample

```diff
@@ -0,0 +1,16 @@
+# These modules are required to enable AT86RF230 802.15.4 radio in the
+# MinnowBoard MAX
+#
+# To enable loading, link as below:
+#
+# install: ln -s /lib/modprobe.d/spi-minnow-at86rf230.conf.sample /etc/modprobe.d/spi-minnow-at86rf230.conf
+# to-autoload: ln -s /usr/lib/modules-load.d/at86rf230.conf.sample /etc/modules-load.d/at86rf230.conf
```

Comment:
well, first there is consistency, if we write tools to do "hwenable spi-minnow-cc2520" it would be simpler to implement.

Also, AFAIU the other modules loaded they are because of spi-minnow-cc2520... so that's the main module to load.